### ISSUE TITLE
Problem running training_unsupervised_seg.py / missing package used in test_unsupervised_seg.py?

### ISSUE THREAD

**Issue by Stelioschatzichronis** [voxelmorph/voxelmorph#270]
**Task**
Run the script 'training_unsupervised_seg.py'.

In kaggle I get the Error:
![image](https://user-images.githubusercontent.com/52251860/107493688-9391bb80-6b96-11eb-877b-f393fd5686b0.png)

In PC l had the error : 
![image](https://user-images.githubusercontent.com/52251860/107494368-5ed23400-6b97-11eb-90ca-6deab7fe5666.png)

I had batch size=1 (Default) 
I changed the lines in files "train_unsupervised_seg.py" and "voxelmorph/networks.py"
"import tensorflow as tf"
to
"import tensorflow.compat.v1 as tf
tf.disable_v2_behavior()"
in order to have compatibility . I only had a few warnings of deprecation

**What have I tried**
I tried this command in kaggle
run ../input/test-sc/train_unsupervised_seg.py ../input/npz-files --atlas ../input/atlas-input/prob_atlas.npz

I had crop the images into (192,160,224) as mentioned and normalized the intensity values by divinding via 255.
Later I have cropped both images and atlas to (160,160,160) but I had the same errors.

**Details of experiments**
I run both in kaggle (in GPU accelerator) and in my PC (CPU, i dont have nvidia GPU) 

kaggle:
GPU 16 GB
CPU 12 GB
Tensorflow 2.4.0

Pc:
CPU 16 GB
Tensorflow 2.4.1
keras 2.3.1

------------------------------------------------------------------------------------------------------
Have I done something wrong? If not, I could find a trained model to import to test_unsupervised_seg.py. In older 
versions of script it is mentioned a "../models/unified_hardcodedstats_lambda_0.01_lr_0.001/60.h5". Is it available on
kaggle? 
Also, in test_unsupervised_seg.py the line 89 
" warped = vxm.tf.ne.transform(input_atlas, input_flow, interp_method='linear', indexing='ij')" 
seems wrong as there is no ne module in vxm.

**Comment by StylianosChatzichronis** [comment]
I run it for (96,80,112) and it worked, it has been a memory issue after all. I had to run separate runs for each part of the 3d image
though. If I have done it correctly, later in test_unsupervised_seg with the editing in line 89 and alteration of labels 11->4. I run it and I had the nifti file. I don't know if mapping file should be reduced from 41 to 11 labels though and if yes what I should use from them.

**Comment by StylianosChatzichronis** [comment]
I have adjusted the atlas in order to fit the new reduced size of the input images and it worked. However, there are still two problems; 
Are the authors have a trained model mentioned in the publication? If there is not something available, can something be done with the baseline in order to use it in my trained model for evaluation?
Also, I the probabilitic atlas has 40 labels but I cannot distinguish the brain regions that each one corresponds. I can only see it when displaying the segmented image. I searched for atlases protocols but yours seem to be customized. Is there any file that describes which brain region each label corresponds?

**Comment by adalca** [comment]
@StylianosChatzichronis  I put these two files at https://surfer.nmr.mgh.harvard.edu/ftp/data/voxelmorph/label_files/, including one to transform from 41 to FS labels, and one for FS labels meaning. Does this help?

**Comment by StylianosChatzichronis** [comment]
It definitely helps. Thank you very much.
